web/src/components/ShareMemoDialog.tsx

```diff
@@ -179,6 +154,31 @@ const ShareMemoDialog: React.FC<Props> = (props: Props) => {
             </button>
           </div>
         </div>
+        <div className="memo-container" ref={memoElRef}>
+          <div className="watermark-container">
+            <div className="logo-container">
+              <img className="h-10 w-auto rounded-lg" src={`${systemStatus.customizedProfile.logoUrl || "/logo.webp"}`} alt="" />
+            </div>
+            <div className="userinfo-container">
+              <span className="name-text">{user.nickname || user.username}</span>
+              <span className="usage-text">
+                {state.memoAmount} MEMOS / {createdDays} DAYS
+              </span>
+            </div>
+            <QRCodeSVG
+              value={`${window.location.origin}/m/${memo.id}`}
+              size={40}
+              bgColor={"#F3F4F6"}
+              fgColor={"#4B5563"}
+              includeMargin={false}
+            />
+          </div>
+          <span className="time-text">{memo.createdAtStr}</span>
+          <div className="memo-content-wrapper">
+            <MemoContent content={memo.content} showFull={true} />
+            <MemoResources resourceList={memo.resourceList} />
+          </div>
```

Comment:
Is this expected? The watermark is on top of content.
<img width="424" alt="image" src="https://user-images.githubusercontent.com/24653555/230000623-4183b208-06d6-4560-8c27-0cdf678a138b.png">

Reply:
Yea I moved it so for really long content, if you simply want to do image or scan qr code it can be on the top without scrolling down. Let me know what you think.

Reply:
The content is more important than the QR code. Maybe we should keep the origin order in this PR and think of a better layout style.

Reply:
To confirm, you mean no change to ShareMemoDialog at all, or still keep the first row "visibility, image, link"?

Reply:
Yeah